docs/streams.html

```diff
@@ -398,6 +401,84 @@
+            </ul>
+        </li>
+        <li> Parameter updates in <code>StreamsConfig</code>:
+            <ul>
+                <li> parameter "zookeeper.connect" was deprecated </li>
+                <li> added many new parameters for metrics, security, and client configurations </li>
+            </ul>
+        </li>
+        <li> Changes in <code>StreamsMetrics</code> interface:
+            <ul>
+                <li> removed methods: <code>#addLatencySensor()</code> </li>
+                <li> added methods: <code>#addLatencyAndThroughputSensor()</code>, <code>#addThroughputSensor()</code>, <code>#recordThroughput()</code>,
+                     <code>#addSensor()</code>, <code>#removeSensor()</code> </li>
+            </ul>
+        </li>
+        <li> New methods in <code>TopologyBuilder</code>:
+            <ul>
+                <li> added overloads for <code>#addSource()</code> that allow to define a <code>"auto.offset.reset"</code> policy per source node </li>
+                <li> added methods <code>#addGlobalStore()</code> to add global <code>StateStore</code>s </li>
+            </ul>
+        </li>
+        <li> New methods in <code>KStreamBuilder</code>:
+            <ul>
+                <li> added overloads for <code>#stream()</code> and <code>#table()</code> that allow to define a <code>"auto.offset.reset"</code> policy per input stream/table </li>
+                <li> <code>#table()</code> always requires to specify a store name </li>
+                <li> added method <code>#globalKTable()</code> to create a <code>GlobalKTable</code> </li>
+            </ul>
+        </li>
+        <li> New joins for <code>KStream</code>:
+            <ul>
+                <li> added overloads for <code>#join()</code> to join with <code>KTable</code> </li>
+                <li> added overloads for <code>#join()</code> and <code>leftJoin()</code> to join with <code>GlobalKTable</code> </li>
+            </ul>
+        </li>
+        <li> Aligned <code>null</code>-key handling for <code>KTable</code> joins:
+            <ul>
+                <li> like all other KTable operations, <code>KTable-KTable</code> joins do not throw an exception on <code>null</code> key records anymore, but drop those records silently </li>
+            </ul>
+        </li>
+        <li> New window type <em>Session Windows</em>:
+            <ul>
+                <li> added class <code>SessionWindows</code> to specify session windows </li>
+                <li> added overlaods for <code>KGroupedStream</code> methods <code>#count()</code>, <code>#reduce()</code>, and <code>#aggregate()</code>
+                     to allow session window aggregations </li>
+            </ul>
+        </li>
+        <li> Changes to <code>TimestampExtractor</code>:
+            <ul>
+                <li> method <code>#extract()</code> has a second parameter now </li>
+                <li> new default timestamp extractor class <code>FailOnInvalidTimestamp</code> </li>
```

Comment:
We should a few words that

1. describe how the new default works
2. how the new default compares to the previous default
3. how the user can, if needed, revert back to the previous (default) behavior